app_legacy/Helpers/database/user-activity.php

```diff
@@ -506,8 +506,9 @@ function getLatestRichPresenceUpdates(): array
               LEFT JOIN GameData AS gd ON gd.ID = ua.LastGameID
               LEFT JOIN Console AS c ON c.ID = gd.ConsoleID
               WHERE ua.RichPresenceMsgDate > TIMESTAMPADD( MINUTE, -$recentMinutes, NOW() )
-                AND ua.LastGameID !=0
+                AND ua.LastGameID != 0
                 AND ua.Permissions >= $permissionsCutoff
+                AND NOT ua.Untracked
```

Comment:
I feel that untracked users should still appear in the active players list. Untracking just removes them from the leaderboards, they still appear in recent activity for individual games.

Could you wrap RAPoints and RASoftcorePoints such that they return 0 for untracked users?

Reply:
Good idea, I've removed the `AND NOT` clause and wrapped the `RAPoints` and `RASoftcorePoints` in `IF()`s such that if the user is untracked, these values will return 0.